vector/src/main/java/im/vector/app/features/location/live/map/LocationLiveMapViewFragment.kt

```diff
@@ -16,24 +16,40 @@
 
 package im.vector.app.features.location.live.map
 
+import android.graphics.drawable.Drawable
 import android.os.Bundle
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import androidx.core.graphics.drawable.toBitmap
 import androidx.lifecycle.lifecycleScope
 import com.airbnb.mvrx.args
+import com.airbnb.mvrx.fragmentViewModel
+import com.airbnb.mvrx.withState
+import com.mapbox.mapboxsdk.geometry.LatLng
+import com.mapbox.mapboxsdk.geometry.LatLngBounds
+import com.mapbox.mapboxsdk.maps.MapView
+import com.mapbox.mapboxsdk.maps.MapboxMap
 import com.mapbox.mapboxsdk.maps.MapboxMapOptions
+import com.mapbox.mapboxsdk.maps.Style
 import com.mapbox.mapboxsdk.maps.SupportMapFragment
+import com.mapbox.mapboxsdk.plugins.annotation.SymbolManager
+import com.mapbox.mapboxsdk.plugins.annotation.SymbolOptions
+import com.mapbox.mapboxsdk.style.layers.Property
 import dagger.hilt.android.AndroidEntryPoint
 import im.vector.app.R
 import im.vector.app.core.extensions.addChildFragment
 import im.vector.app.core.platform.VectorBaseFragment
 import im.vector.app.databinding.FragmentSimpleContainerBinding
 import im.vector.app.features.location.UrlMapProvider
+import im.vector.app.features.location.zoomToBounds
+import im.vector.app.features.location.zoomToLocation
+import timber.log.Timber
+import java.lang.ref.WeakReference
 import javax.inject.Inject
 
 /**
- * Screen showing a map with all the current users sharing their live location in room.
+ * Screen showing a map with all the current users sharing their live location in a room.
  */
 @AndroidEntryPoint
```

Comment:
We should use constructor injection and also inject UrlMapProvider there.

Reply:
Done.

Reply:
@onurays In a previous PR I had a [comment about using `@AndroidEntryPoint`](https://github.com/vector-im/element-android/pull/6092#discussion_r876857466) to inject fragment. And I think this is a better way to inject fragment.

Reply:
@ganfra is the master of the Dagger. Feel free to revert :)